htdocs/projet/tasks/task.php

```diff
@@ -205,7 +205,15 @@
 
 	$langs->load("other");
 	$upload_dir = $conf->project->dir_output;
-	$file = $upload_dir.'/'.dol_sanitizeFileName(GETPOST('file'));
+	//in case of a task filename is like PJ2211-0001%2FTK2211-0001%2FTK2211-0001_task.odt
+	//then dol_sanitizeFileName will convert it to PJ2211-0001_FTK2211-0001_FTK2211-0001_task.odt
+	//and that file does not exists so can't be deleted
+	$sanitizedFileName = dol_sanitizeFileName(GETPOST('file'));
```

Comment:
This fix is very suspicious. If we have to manipulate the parameter fil to get the correct path of file, the problem is probably in the link where parameter file is wrong.

Reply:
of course but tasts are a the only case where files are in a subdirectory ... so please let me know what kind of parameter we could make to do it better ?

`.../...file=PJ2211-0001/FTK2211-0001/FTK2211-0001_task.odt.../...`

must be replaced by what ?